src/System.Runtime/tests/System/Text/StringBuilderTests.cs

```diff
@@ -1086,26 +1086,19 @@ public static IEnumerable<object[]> Equals_TestData()
 
             var sb4 = new StringBuilder(10, 20);
             var sb5 = new StringBuilder(10, 20);
-            var sb6 = new StringBuilder(10, 21);
-            var sb7 = new StringBuilder(11, 20);
 
-            var sb8 = new StringBuilder(10, 20);
-            sb8.Append("Hello");
-            var sb9 = new StringBuilder(10, 20);
-            sb9.Append("Hello");
-            var sb10 = new StringBuilder(10, 20);
-            sb10.Append("HelloX");
+            var sb6 = new StringBuilder(10, 20).Append("Hello");
+            var sb7 = new StringBuilder(10, 20).Append("Hello");
+            var sb8 = new StringBuilder(10, 20).Append("HelloX");
```

Comment:
Should you have a test case with the same content with different capacity to validate that capacity doesn't not influence Equals?

Reply:
Yes, there's a PR out with additional tests to validate that.  I put this PR up with the deletions to allow the coreclr auto-update PR to go through, as it's currently failing due to these tests.